test/e2e/framework/pv_util.go

```diff
@@ -233,15 +290,18 @@ func CreatePVCPV(c clientset.Interface, pvConfig PersistentVolumeConfig, pvcConf
 
 	By(fmt.Sprintf("Creating a PVC followed by a%s PV", preBindMsg))
 	// instantiate the pvc
-	pvc = CreatePVC(c, ns, pvc)
+	if pvc, err = CreatePVC(c, ns, pvc); err != nil {
```

Comment:
ack. refactored _pv_util.go_ to minimize explicit declaration of vars, and to not use the compound `if func-call(...); err != nil {` construct.